src/amp-story-player/amp-story-player-impl.js

```diff
@@ -87,6 +87,17 @@ const BUTTON_EVENTS = {
   [BUTTON_TYPES.CLOSE]: 'amp-story-player-close',
 };
 
+/** @enum {string} */
+const STORY_STATE_TYPE = {
```

Comment:
Can we reuse the `enum` below?

Reply:
So I wasn't sure about what publishers should use to specify what state they wanted. I didn't want to expose `PAGE_ATTACHMENT_STATE`. It seemed weird that they would call `getStoryState('PAGE_ATTACHMENT_STATE')` over `getStoryState('page-attachment')`.

Reply:
Is providing the "mute" state also a desired thing? I think if the `STORY_MESSAGE_STATE_TYPE` enum is to be exposed, everything in it should get a valid response from `this.getStoryState` below.

Reply:
So I wasn't sure if there would be more story states exposed, hence the `switch` statement. I don't think there's a need for providing the `mute` state because we have the `mute` and `unmute` methods.